data/172/943/100/1/1729431001.geojson

```diff
@@ -16,6 +16,7 @@
     "mps:longitude":-85.295705,
     "mz:hierarchy_label":1,
     "mz:is_current":1,
+    "mz:min_zoom":30.0,
```

Comment:
What's the rational for this high `min_zoom`? That would never show on the map.

Reply:
This applies to quite a few of these updates.

Reply:
Ah, that was to address this comment: https://github.com/whosonfirst-data/whosonfirst-data-admin-us/pull/86#issuecomment-726291483

Reply:
And combined with `"wof:statistical_gore":1`.